src/esp/gfx/test/DepthUnprojectionTest.cpp

```diff
@@ -25,6 +25,9 @@
 namespace Cr = Corrade;
 namespace Mn = Magnum;
 
+using Mn::Math::Literals::operator""_degf;
+using Mn::Math::Literals::operator""_rgbf;
```

Comment:
I still don't agree with Tidy about this one, but I guess we can't do any better. 

Found https://reviews.llvm.org/D33010 that makes the check not complain for standard literals, but such hardcoded unextendable whitelisting makes me kinda angry :/